docs/docs/community/release-process.md

```diff
@@ -0,0 +1,68 @@
+---
+
+# The Nebari release process
+
+This page describes the high-level details of cutting a new Nebari release.
+
+|        Cadence        |               Versioning System                |                                                                                    Release Checklist                                                                                     |                                                                                        Testing Checklist                                                                                         |
+| :-------------------: | :--------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| 3rd week of the month | [CalVer](#calver-details) - (i.e. `2022.10.1`) | [Link to issue template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=release-checklist.md&title=%5BRELEASE%5D+%3Cversion%3E) | [Link to issue template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=testing-checklist.md&title=Testing+checklist+for+%3Cversion%3E) |
+
+## Release Captain responsibilities
+
+For every release, there is an assigned "Release Captain". The Release Captain's responsibilities are:
+
+- Manage both the release and testing processes and update the checklists as needed.
+- Communicate the status of the release to the rest of the Nebari development team and the community (through updating the checklists and adding status updates as comments).
+- Assign owners to checklist items (if not owned by the Release Captain).
+- Adjust the schedule, particularly the publishing dates, based on defects found, fixes made, holidays, vacations, and so on.
+- Ensure a new branch and milestone is created after completing a release cycle.
+
+## CalVer details
+
+Nebari releases should follow the following CalVer versioning style:
+
+```
+YYYY-MM-releaseNumber
+```
+
+:::info
+`YYYY` represents the year - i.e. `2023`
+
+`MM` represents the _non-zero padded_ month - i.e. `1` for January, `12` for December
+
+`releaseNumber` represents the current release for that month, starting at `1`. Anything above `1` represents a [hotfix (patch) release](#hotfixes).
+:::
+
+:::caution
+For the release tag, there should be NO prepended `v`
+:::
+
+For example, the first Nebari CalVer release was `2022.10.1`. If a hotfix release was needed in the same month, we increment the `releaseNumber` by 1, which would be `2022.10.2` (_this is to illustrate how the increment works, this release does not exist._)
+
+## Gitflow and branching strategy
```

Comment:
This isn't very clear - as Gitflow is a branching/contribution strategy and we are not using Gitflow (well not strict gitflow anyway)
```suggestion
## Branching strategy
```